airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/MssqlSourceTest.java

```diff
@@ -108,6 +114,9 @@ void testDiscoverWithPk() throws Exception {
 
     final AirbyteCatalog actual = new MssqlSource().discover(config);
     assertEquals(CATALOG, actual);
+
+    List<AirbyteMessage> read = new MssqlSource().read(config, CatalogHelpers.toDefaultConfiguredCatalog(actual), null).collect(Collectors.toList());
```

Comment:
what is this doing / testing?

Reply:
oh crap. I accidentally added this to the diff.